test/PubSub/PublishedMessageTest.cs

```diff
@@ -1,35 +1,35 @@
-﻿using Ipfs;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using ProtoBuf;
-using System;
+﻿using System;
 using System.IO;
 using System.Linq;
-using System.Threading;
-using System.Threading.Tasks;
+using Ipfs;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Google.Protobuf;
 
 namespace PeerTalk.PubSub
 {
-    
+
     [TestClass]
     public class PublishedMessageTest
     {
-        Peer self = new Peer { Id = "QmXK9VBxaXFuuT29AaPUTgW3jBWZ9JgLVZYdMYTHC6LLAH" };
-        Peer other = new Peer { Id = "QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ" };
+        private readonly Peer self = new Peer { Id = "QmXK9VBxaXFuuT29AaPUTgW3jBWZ9JgLVZYdMYTHC6LLAH" };
+        private readonly Peer other = new Peer { Id = "QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ" };
 
         [TestMethod]
         public void RoundTrip()
         {
             var a = new PublishedMessage
             {
-                Topics = new string[] { "topic" },
                 Sender = self,
-                SequenceNumber = new byte[] { 1, 2, 3, 4, 5, 6, 7, 8},
-                DataBytes = new byte[] { 0, 1, 0xfe, 0xff }
+                SequenceNumberProto = ByteString.CopyFrom(new byte[] { 1, 2, 3, 4, 5, 6, 7, 8 }),
+                DataBytesProto = ByteString.CopyFrom(new byte[] { 0, 1, 0xfe, 0xff }),
```

Comment:
Cannot change the names of public members  SequenceNumber and DataBytes must accept byte array.

Why is ByteString used?

Reply:
type bytes in .proto file is represented as ByteString in generated c# code

Reply:
I can do some trick to the extended partial class to make the API compatible, in the version being discussed,  SequenceNumber and DataBytes are get only properties